test/__snapshots__/autocomplete.test.ts.snap

```diff
@@ -20,7 +20,7 @@ exports[`autocomplete > should accept variants 1`] = `
 
 exports[`autocomplete > should provide autocomplete 1`] = `
 {
-  "align-": "align-base align-baseline align-bottom align-btm align-end align-inherit align-initial align-mid align-middle align-revert",
```

Comment:
Why removing this? These values are valid.

Reply:
Not removed.😂